lib/payment_processor/go_cardless/populator.rb

```diff
@@ -25,12 +25,9 @@ def subscription_params
         request_params.merge(
           {
             name: "donation",
-            interval_unit: "monthly"
-          }.tap do |params|
-            if bacs?
-              params[:start_date] = Helper.next_available_date(Settings.gocardless.gbp_charge_day.to_i)
-            end
-          end
+            interval_unit: "monthly",
+            start_date: charge_date
```

Comment:
This is the only difference I can see between transactions and subscriptions, and transactions are working fine.